tools/compileProtocol.js

```diff
@@ -1,56 +1,19 @@
 /**
- * This is a utility script that converts the YAML here into ProtoDef schema code and (soon) docs/typescript definitions.
- * It also pre-compiles JS code from the schema for easier development.
- *
+ * Pre-compiles JS code from the schema for easier development.
  * You can run this with `npm run build`
- *
  */
 const fs = require('fs')
 const { ProtoDefCompiler } = require('protodef').Compiler
-const { Versions } = require('../src/options')
+const { convert } = require('minecraft-data/minecraft-data/tools/js/compileProtocol')
```

Comment:
what about putting that somewhere where it makes more sense ? maybe protodef yaml ? (might be needed to make it slightly more generic, so it makes sense to put it there)

Reply:
Removed most of the references to this. This is a utility script to generate JS from the yml files, I use it primarily for debugging (e.g. easier to insert logging or other things into serialization code), but we can remove it from the repo if needed (the code otherwise will compile the JSON at runtime).